### PR TITLE
Create web3.md

### PR DESCRIPTION
**Pull request by monicazng**
To be REVIEWED before merging

### CODE REVIEW

File: docs/jobs/openings/web3.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+## Web3 Developer Opening at Aragon
+
+Aragon is an open source, decentralized project fighting for the freedom to organize. We believe that decentralized organizations can solve the world’s worst problems.
+
+We’re looking for a Web3 Developer to help us consolidate smart contracts with a frontend, we call this position The Glue. Our Web 3 Developer must not only be a Javascript rockstar but also a decentralization evangelist, someone who enjoys researching and testing new Web3 technologies.
```
Comment (sohkai) [inline docs/jobs/openings/web3.md]: I'd rephrase to:

> We’re looking for a Web3 Developer to help us **bridge** smart contracts **to** a frontend. **Internally, we call this position "Glue".** Our Web3 Developer **is ideally** not only a Javascript expert but also a decentralization evangelist—someone who enjoys researching and testing new Web3 technologies.

File: docs/jobs/openings/web3.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+### Responsibilities
+
+- Take ownership of the development and maintenance of [aragon.js](https://github.com/aragon/aragon.js), the JavaScript library between smart contracts and the native web experience.
```
Comment (sohkai) [inline docs/jobs/openings/web3.md]: the JavaScript ~~library~~ interface

File: docs/jobs/openings/web3.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+- Research and experiment with new Web3 technologies, including but not limited to [Web3.js](https://github.com/ethereum/web3.js/) and [Parity systems](https://github.com/paritytech).
```
Comment (sohkai) [inline docs/jobs/openings/web3.md]: I'd remove Paritytech and include [ethers.js](https://github.com/ethers-io/ethers.js/), [Parity's 007 Bonds](https://github.com/paritytech/oo7-parity) and [ethjs](https://github.com/ethjs/ethjs) for good measure.

File: docs/jobs/openings/web3.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+- Assist on smart contract and frontend development when needed.
+- Help create developer tools and technical documentation.
```
Comment (sohkai) [inline docs/jobs/openings/web3.md]: I'd remove this and the point above to instead add "Assist others with Web3-related development when needed" to broaden the scope to be encompassing all things related to Web3.

File: docs/jobs/openings/web3.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+### Requirements
+
+- Track record working with Javascript environments.
+- Expert understanding on ES6/ES7, React, client-side storage, resource optimization, CSS, using processors such as LESS or SASS, RxJS, service workers, and device APIs.
```
Comment (sohkai) [inline docs/jobs/openings/web3.md]: I'd remove the `React` and `CSS, using processors such as LESS or SASS`, and `device APIs` bits, to replace them with `functional programming` and `Redux / Elm-inspired architectures`.

File: docs/jobs/openings/web3.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+
+### Good to have
+
+- Previous experience contributing or managing open source projects.
+- Previous experience working remotely.
+- Very good written communication skills.
+
+### How to apply
+
+- Write an email with the following attachments plus any information you consider important to *hiring@aragon.one*, with the subject `Web3 Developer Dev opening application`.
```
Comment (sohkai) [inline docs/jobs/openings/web3.md]: `Web3 Developer Dev` (Dev seems a little redundant)

File: docs/jobs/openings/web3.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+- Please send along your GitHub profile. Since we are open source only, we require applicants for developer positions to have a very active and healthy GitHub profile.
```
Comment (sohkai) [inline docs/jobs/openings/web3.md]: > we require applicants for developer positions to have a very active and healthy GitHub profile.

This is probably on all our positions, but I feel like we could relax this a little bit. Being excited by open source is not the same as having the opportunities / luxuries to work on a "healthy" Github profile. It's already a "good to have", which is enough, I think.

It'll be very obvious interacting with the person if they've never heard of Github or used anything open source.

File: docs/jobs/openings/web3.md (hunk 2 above)
Comment (Smokyish) [reply docs/jobs/openings/web3.md]: Should we then also change the repo description, since that says `A JavaScript library for interacting with Aragon`?

File: docs/jobs/openings/web3.md (hunk 1 above)
Comment (onbjerg) [reply docs/jobs/openings/web3.md]: To be honest, "a frontend" also sounds super vague, either do "frontends" or "the frontend" (I'd suggest frontends since we're talking about multiple representations, not just aragon/aragon)

File: docs/jobs/openings/web3.md (hunk 2 above)
Comment (onbjerg) [reply docs/jobs/openings/web3.md]: I'd keep interface, the difference is negligible anyway and it is, in fact, a library

File: docs/jobs/openings/web3.md (hunk 5 above)
Comment (onbjerg) [reply docs/jobs/openings/web3.md]: The Web3 developer is probably not going to spend a lot of time using CSS and we haven't used a single device API, so I agree with Brett.

File: docs/jobs/openings/web3.md (hunk 7 above)
Comment (onbjerg) [reply docs/jobs/openings/web3.md]: Yeah, as long as they have something to show, too, it won't matter too much. We have to make sure we're not excluding people based on how much time their previous job allowed them to work on open source :)

File: docs/jobs/openings/web3.md (hunk 5 above)
Comment (Smokyish) [reply docs/jobs/openings/web3.md]: That description is very similar to the frontend position listing, on that, the description was changed to

> Modern JavaScript (ES6+) and Web APIs, CSS, React (Web and Native), resource optimization, development tools.

Like the new suggestions :+1:

Comment (monicazng) [reply docs/jobs/openings/web3.md]: I was specially confused on this end, so thanks for the input!

File: docs/jobs/openings/web3.md (hunk 7 above)
Comment (monicazng) [reply docs/jobs/openings/web3.md]: Totally agree. Furthermore, I think we should update this line on all other job postings. It was inherited from the old job description and the truth is that most people don't really have healthy Github profiles.

**Comment by Smokyish** [comment]
@sohkai @onbjerg @luisivan Can we get at least one approving review for merging?

File: docs/jobs/openings/web3.md [hunk 8]
```diff
@@ -0,0 +1,33 @@
+## Web3 Developer Opening at Aragon
+
+Aragon is an open source, decentralized project fighting for the freedom to organize. We believe that decentralized organizations can solve the world’s worst problems.
+
+We’re looking for a Web3 Developer to help us bridge smart contracts to a frontend. Internally, we call this position "Glue". Our Web3 Developer is ideally not only a Javascript expert but also a decentralization evangelist—someone who enjoys researching and testing new Web3 technologies.
+
+### Responsibilities
+
+- Take ownership of the development and maintenance of [aragon.js](https://github.com/aragon/aragon.js), the JavaScript interface between smart contracts and the native web experience.
+- Research and experiment with new Web3 technologies, including but not limited to [Web3.js](https://github.com/ethereum/web3.js/), [ethers.js](https://github.com/ethers-io/ethers.js/), [Parity's 007 Bonds and [ethjs](https://github.com/paritytech/oo7-parity) and [eth.js](https://github.com/ethjs/ethjs).
```
Comment (sohkai) [inline docs/jobs/openings/web3.md]: One thing: I think this is broken; it should just be:

`[Parity's 007 Bonds](https://github.com/paritytech/oo7-parity), and [eth.js](https://github.com/ethjs/ethjs).`
